crates/oxc_linter/src/snapshots/no_mixed_operators.snap

```diff
@@ -0,0 +1,126 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 53
+expression: no_mixed_operators
+---
+
+  × eslint(no-mixed-operators): Disallow mixed binary operators
+   ╭─[no_mixed_operators.tsx:1:1]
+ 1 │ a && b || c
+   ·   ─┬   ─┬
+   ·    │    ╰──  with ||. Use parentheses to clarify the intended order of operations.
+   ·    ╰── Unexpected mix of &&
```

Comment:
You can put "Use parentheses to clarify the intended order of operations." into the help section.

And I think the labels on the two spans can be removed, it's hard to put messages on think and make them readable.

Reply:
I didn't realize the error message can reference the operator arguments. I moved the message there. Hopefully it's more readable now